<ui version="4.0" >
 <class>helpDialog</class>
 <widget class="QDialog" name="helpDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RealSurf Help</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resource.qrc" >:/resources/help.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">RealSurf Help&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="helpBrowser" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>190</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>190</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="tabStopWidth" >
      <number>0</number>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_55" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>button_55</sender>
   <signal>accepted()</signal>
   <receiver>helpDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_55</sender>
   <signal>rejected()</signal>
   <receiver>helpDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
